internal/pkg/throttle/throttle_test.go

```diff
@@ -17,7 +17,7 @@ func TestThrottleZero(t *testing.T) {
 	// but still cannot acquire existing that has not timed out
 	throttle := NewThrottle(0)
 
-	N := rand.Intn(2048) + 10
+	N := rand.Intn(64) + 10
```

Comment:
This is related to the cache changes? Just curious.

Reply:
Logger appears to be running at trace level during tests, which generates a  lot of noise.  The large numbers here are unnecessary to demonstrate the logic so I tuned it down a bit.